src/orion/core/utils/__init__.py

```diff
@@ -27,6 +27,15 @@ def nesteddict():
     return defaultdict(nesteddict)
 
 
+def get_qualified_name(package, name):
```

Comment:
`_get_qualified_name`

Reply:
I'm tempted to leave it public just in case it is needed elsewhere